.gitignore

```diff
@@ -3,6 +3,9 @@ syntax: glob
 # Build tools related files
 /[Tt]ools/
 
+# CoreRT ignores
+.vcvarsallwithargs
```

Comment:
The temp files should be under `IntermediateOutputPath` so that this is not necessary.